go/internal/feast/server/http_server.go

```diff
@@ -136,6 +136,7 @@ type getOnlineFeaturesRequest struct {
 	Entities         map[string]repeatedValue `json:"entities"`
 	FullFeatureNames bool                     `json:"full_feature_names"`
 	RequestContext   map[string]repeatedValue `json:"request_context"`
+	Status           bool                     `json:"status"`
```

Comment:
Instead of this can we use query parameters? It may confuse users if they are using feast documentation and create request. @vinaymarpina What do you think about it?

Reply:
Do we want to have separate query params for status and event timestamps? Imagine it would be confusing for users if they omit status and also don't get timestamps.
Also, I feel we should include default values for these parameters. Think we should default them to be on.

Reply:
Middleware is the client for this. It won't impact any SDK users. So we thought it would be good to handle both with only one query parameter. We are defaulting it to False. Otherwise there is work involved in Middleware's.

Reply:
I think its good for us to default to False at this stage and clear in the documentation so that customers can opt True if they see latency concerns.